### PR TITLE
chore: use docker on win and catalina

### PR DESCRIPTION
**Pull request by endarova**
Add methods to start and stop docker on windows and macos Catalina
Start and stop docker in setup and tear down class of sets that build with snapshot.
Remove skip of tests with snapshot on win and catalina

### CODE REVIEW

File: core/utils/docker.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+from core.settings import Settings
+from core.enums.os_type import OSType
+
+
+class Docker(object):
+
+    @staticmethod
+    def start():
+        if Settings.HOST_OS == OSType.WINDOWS:
+            cmd = r'"C:\Program Files\Docker\Docker\Docker Desktop.exe"'
```
Comment (dtopuzov) [inline core/utils/docker.py]: May we do it this way:
- Have DOCKER_PATH in settings
- Read it from env variable and fallback to this value as default if variable is not set (on my machine path is different).

Comment (endarova) [reply core/utils/docker.py]: Agree, will do it this way

Comment (dtopuzov) [reply core/utils/docker.py]: May be also ok to read env varialbe in this file, just to have the variable somewhere.

**Comment by endarova** [comment]
run core tests

**Pull request merged**